shared/src/commonMain/kotlin/data/credentials/CertificateOfResidenceCredentialAdapter.kt

```diff
@@ -136,6 +136,9 @@ sealed class CertificateOfResidenceCredentialAdapter : CredentialAdapter() {
 private class CertificateOfResidenceCredentialSdJwtAdapter(
     private val attributes: Map<String, JsonPrimitive>,
 ) : CertificateOfResidenceCredentialAdapter() {
+    override val scheme: ConstantIndex.CredentialScheme
```

Comment:
please elaborate why this and the next change is needed, so I don't have to dig into it myelf

Reply:
So that we know in other methods, if this is the "old" `EuPidScheme` or the "new" `EuPidSdJwtScheme`.